app/models/beer.rb

```diff
@@ -24,4 +24,12 @@ def self.search_message
       "No Matching Beer"
     end
   end
+
+  def average_rating
+    if reviews.average(:rating) == nil
+      reviews.average(:rating)
```

Comment:
But this is why I can't do rounding in the view, unless I bring the if/else statements with it